launcher.py

```diff
@@ -48,6 +117,9 @@ def compile_application(self) -> bool:
     def deploy_contract(self) -> bool:
         ''' Deploy the smart contract and populate the ~/config.exs file
         '''
+        if self.ethereum_network == [x for x in self.public_networks]:
```

Comment:
Shouldn't it be `if self.ethereum_network in self.public_networks:`?

Reply:
Thanks, that was a bug. Resolved in 41f2545